app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,67 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
Code in  ManageIQ::Providers::ContainerManager is for *any* container manager...it could even be something like docker swarm in the future.